<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dobavit_comand</class>
 <widget class="QDialog" name="dobavit_comand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Добавить строку в таблицу</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>обновить IDсоревнований</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Вид спорта</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Команда</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Команда</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Название соревнований</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Тренер</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
